automatic/ffmpeg/tools/chocolateyInstall.ps1

```diff
@@ -1,14 +1,16 @@
 ﻿$ErrorActionPreference = 'Stop'
 
 $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
-$embedded_path = if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
-         Write-Host "Installing 64 bit version"; gi "$toolsPath\*_x64.zip"
-} else { Write-Host "Installing 32 bit version"; gi "$toolsPath\*_x32.zip" }
 
 $packageArgs = @{
-  PackageName    = 'ffmpeg'
-  FileFullPath   = $embedded_path
-  Destination    = $toolsPath
+    PackageName    = ''
+    FileFullPath   = gi $toolsPath\*_x32.zip
+    FileFullPath64 = gi $toolsPath\*_x64.zip    
+    Destination    = $toolsPath
 }
+
+ls $toolsPath\* | ? { $_.PSISContainer } | rm -Recurse -Force #remove older package dirs
 Get-ChocolateyUnzip @packageArgs
 rm $toolsPath\*.zip -ea 0
+
+mv $toolsPath\ffmpeg-* $toolsPath\ffmpeg
```

Comment:
I think this could cause problem with users that have the ffmpeg package already installed, but meh guess we'll handle that then.